fastify.js

```diff
@@ -133,7 +133,9 @@ function fastify (options) {
   // Default router
   const router = buildRouting({
     config: {
-      defaultRoute: defaultRoute,
+      defaultRoute: function (req, res) {
```

Comment:
This change doesn't do anything, does it?

Reply:
I _think_ this is needed to create a level of indirection, so that somebody can alter `fastify.defaultRoute` and make it work. I don't understand this change.

Reply:
Ah, right, they are different! Thanks!

Reply:
Yes exactly, this indirection is needed because `fastify` passes the `defaultRoute` to the router, which now has a different reference to the function.
When `fastify-websocket` updates it, only the `fastify` reference is changed and not the router's.